### PR TITLE
feat(HeroLocationCtrl) creates list for selecting current location

### PR DESCRIPTION
**Pull request by darrinmn9**

### CODE REVIEW

File: client/src/components/hero_location/heroLocationCtrl.js [hunk 1]
```diff
@@ -2,10 +2,55 @@
...
+      };
+
+      vm.confirm = function(){
+        //set location of hero to vm.locations[vm.selection]
+        heroFactory.setOrder(vm.locations[vm.selection]);
+        $state.go('hero_task');
+      };
+
+
+      function success(position) {
```
Comment (tats-tagawa) [inline client/src/components/hero_location/heroLocationCtrl.js:42]: any reason you changed from function expression to function declaration? Convention?

**Comment by tats-tagawa** [comment]
1. Your git commit message is missing a colon. Since you only have one commit, please do `git commit --amend` and change it.
2. On my end, the top location can't be clicked, I think it is the navbar being on top of it.
3. Update Trello with # included

File: client/src/components/hero_location/heroLocationCtrl.js (hunk 1 above)
Comment (darrinmn9) [inline client/src/components/hero_location/heroLocationCtrl.js:42]: Basically, I use function expressions when binding something to scope. If im using a function that is not bound to scope, i use a function declaration and put it at the bottom of the code.

Lets talk more about this tomorrow

Comment (darrinmn9) [inline client/src/components/hero_location/heroLocationCtrl.js:42]: Also... i totally screwed up this pull request. Was accidentally making changes to heroTaskControl and didnt know how to move my commits to another branch. Then i committed them... and i missed the : on this message as well... ahh goodnight

Comment (darrinmn9) [inline client/src/components/hero_location/heroLocationCtrl.js:42]: ...and by screwed up i mean... the code all works, but i totally screwed up the git workflow. I need to get better at this. LOL

**Pull request merged**